R/FilterState.R

```diff
@@ -149,7 +149,7 @@ FilterState <- R6::R6Class( # nolint
     set_state = function(state) {
       checkmate::assert_class(state, "teal_slice")
       if (private$is_fixed()) {
-        logger::log_warn("attempt to set state on fixed filter aborted id: { private$get_id() }")
+        warning(sprintf("attempt to set state on fixed filter aborted id: %s", private$get_id()))
```

Comment:
`warning` doesn't need `sprintf`
```suggestion
        warning("attempt to set state on fixed filter aborted id: ", private$get_id())
```